frontend/src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -88,6 +88,9 @@ const StyledRoleOption = styled('div')(({ theme }) => ({
     },
 }));
 
+const caseInsensitiveSearch = (search: string, value?: string) =>
+    Boolean(value?.toLowerCase()?.includes(search.toLowerCase()));
+
```

Comment:
I wonder if this should be in some kind of library

Reply:
How about https://github.com/Unleash/unleash/pull/1988/commits/5a1250a8f7f8a4363619cd2d26e1237d0d64a033?

Reply:
Amazing :rocket: